Common subdirectories: pynitrokey-0.4.33.orig/.github and pynitrokey-0.4.33/.github
Common subdirectories: pynitrokey-0.4.33.orig/ci-scripts and pynitrokey-0.4.33/ci-scripts
Common subdirectories: pynitrokey-0.4.33.orig/docs and pynitrokey-0.4.33/docs
Common subdirectories: pynitrokey-0.4.33.orig/pynitrokey and pynitrokey-0.4.33/pynitrokey
diff '--color=auto' -u pynitrokey-0.4.33.orig/pyproject.toml pynitrokey-0.4.33/pyproject.toml
--- pynitrokey-0.4.33.orig/pyproject.toml	2023-03-05 19:28:59.544777594 +0100
+++ pynitrokey-0.4.33/pyproject.toml	2023-03-05 19:48:39.588027456 +0100
@@ -25,19 +25,19 @@
   "click >=8.0.0,<9",
   "cryptography >=3.4.4,<37",
   "ecdsa",
-  "frozendict ~= 2.3.4",
+  "frozendict ~= 2.3",
   "fido2 >=1.1.0,<2",
   "intelhex",
   "nkdfu",
   "nrfutil >=6.1.4,<7",
-  "python-dateutil ~= 2.7.0",
+  "python-dateutil ~= 2.7",
   "pyusb",
   "requests",
-  "spsdk >=1.7.0,<1.8.0",
+  "spsdk >=1.7.0,<=1.9.0",
   "tqdm",
   "urllib3 ~= 1.26.7",
   "tlv8",
-  "typing_extensions ~= 4.3.0"
+  "typing_extensions ~= 4.3"
 ]
 dynamic = ["version", "description"]
 
Common subdirectories: pynitrokey-0.4.33.orig/wine-build and pynitrokey-0.4.33/wine-build
